service/src/main/java/bio/terra/externalcreds/dataAccess/FenceAccountKeyDAO.java

```diff
@@ -32,6 +33,23 @@ public FenceAccountKeyDAO(NamedParameterJdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
+  public Optional<FenceAccountKey> getFenceAccountKey(LinkedAccount linkedAccount) {
```

Comment:
Does this also need a `@WithSpan` annotation? I realized I don't actually know what that does 🤔

Reply:
I believe it does

Reply:
`@WithSpan` allows us to trace the performance of the DB query separate from the API request that triggered it.